code/modules/paperwork/photography.dm

```diff
@@ -484,3 +485,126 @@
 	C.toner -= 20	 //Cyborgs are very ineffeicient at printing an image
 	visible_message("[C.name] spits out a photograph from a narrow slot on its chassis.")
 	usr << "<span class='notice'>You print a photograph.</span>"
+
+// Picture frames
+
+/obj/item/weapon/picture_frame

```

Comment:
update I have tried using the wall frame system and it fucked everything up, the functionality is just way too different